### PR TITLE
Add Image validator helper func

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
> A basic URL verification will be helpful, to validate if the url provided is acutally a uri and not a random text/executable script code. It will also help to skip the effort of trying to load the url if it's invalid.

Great comment!

File: src/custom/Helpers/UrlValidator/image-url-validation.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+};
+
+// Function to verify if URL is valid
+const isUrlValid = (url: string): boolean => {
+  try {
+    new URL(url);
+    return true;
+  } catch (error) {
+    console.error('Invalid URL:', url);
```
Comment (aabidsofi19) [inline src/custom/Helpers/UrlValidator/image-url-validation.tsx:35]: remove the log from here , utils should be pure .

Comment (aabidsofi19) [inline src/custom/Helpers/UrlValidator/image-url-validation.tsx:35]:
```suggestion

```

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]

File: src/custom/Helpers/UrlValidator/image-url-validation.tsx (hunk 1 above)
Comment (leecalcote) [reply src/custom/Helpers/UrlValidator/image-url-validation.tsx]: Accepted suggestion.

**Comment by leecalcote** [comment]
merge conflict...

**Comment by leecalcote** [comment]
I'll offer the suggestion that we close this PR. What do you think, @sudhanshutech?

**Pull request closed without merging**